bitnami/contour/templates/envoy/deployment.yaml

```
@@ -0,0 +1,267 @@
+{{- if .Values.envoy.enabled }}
+{{- if eq .Values.envoy.kind "Deployment" }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "common.names.fullname" . }}-envoy
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: envoy
```

Missing common annotations and labels:
```suggestion
  labels: {{- include "common.labels.standard" . | nindent 4 }}
    app.kubernetes.io/component: envoy
    {{- if .Values.commonLabels }}
    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
    {{- end }}
  {{- if .Values.commonAnnotations }}
  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
  {{- end }}
```